apps/projects/app/components/Panel/FundIssues/FundIssues.js

```diff
@@ -381,16 +388,16 @@ const FundIssues = ({ issues, mode }) => {
     // computes an array of issues and denests the actual issue object for smart contract
     const issuesArray = []
 
-    for (let key in issues) {
+    for (let key in bountylessIssues) {
```

Comment:
While we're here, let's change this to a `map`:

```js
const issuesArray = bountylessIssues.map(issue => {
  const {
    exp,
    fundingHistory,
    deadline,
    hours,
    payout,
    token,
  } = bounties[issue.id]
  return {
    ...issue,
    key: issue.id, // what is this? I think we can probably get rid of it
    exp,
    fundingHistory,
    deadline,
    hours,
    payout,
    token,
  }
})
```

Note that this new logic conflicts with my under-review work at #1659. If #1659 ends up getting merged before this one, I can help you resolve the conflicts. If this one gets merged first, I can handle it!